datafusion/core/tests/fuzz_cases/join_fuzz.rs

```diff
@@ -394,17 +430,31 @@ impl JoinFuzzTestCase {
             let hj = self.hash_join();
             let hj_collected = collect(hj, task_ctx.clone()).await.unwrap();
 
+            let nlj = self.nested_loop_join();
+            let nlj_collected = collect(nlj, task_ctx.clone()).await.unwrap();
+
             // Get actual row counts(without formatting overhead) for HJ and SMJ
             let hj_rows = hj_collected.iter().fold(0, |acc, b| acc + b.num_rows());
             let smj_rows = smj_collected.iter().fold(0, |acc, b| acc + b.num_rows());
+            let nlj_rows = nlj_collected.iter().fold(0, |acc, b| acc + b.num_rows());
+
+            // if debug flag is set the test will save randomly generated inputs and outputs to user folders
+            // so it is easy to run debug on top of the failed test data
+            if debug {
```

Comment:
👍 

It might make sense to add this information about saving information to the overall docstring of this function (`run_tests`) so people don't have to read the implementation to find out what debug does.